webblog/frontend/src/actions/types.js

```diff
@@ -0,0 +1,14 @@
+export const GET_BLOGLIST = "GET_BLOGLIST";
```

Comment:
add separates objects for each type of messages... Like one for Blogs related constants and similarly for others